int32[4] maxBatt
int32[4] curBatt

